### PR TITLE
EMCS 454 - only send milliseconds to EIS, ensure timestamps in tests are accurate

### PR DESCRIPTION
**Pull request by ChrisCookOC**

### CODE REVIEW

File: app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala [hunk 1]
```diff
@@ -143,7 +142,7 @@ class DraftExciseMovementController @Inject()(
     EitherT.fromOption(
       request.headers.get(Constants.XClientIdHeader),
       BadRequest(Json.toJson(ErrorResponse(
-        Instant.now,
+        dateTimeService.timestamp(),
```
Comment (RobertLaverick) [inline app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala:145]: shouldn't this be timestampToMilliseconds?

**Review by RobertLaverick — CHANGES_REQUESTED** [review]

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala [hunk 2]
```diff
@@ -151,8 +154,8 @@ class ExciseNumberQueueWorkItemRepositoryItSpec extends PlaySpec
   private def createWorkItemForErn(ern: String, updateAt: Instant = timestamp)  =
```
Comment (mauro-fanton) [inline it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala:154]: updateAt argument is not used, can we remove it please? I know this was already there,

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala [hunk 3]
```diff
@@ -125,16 +127,17 @@ class ExciseNumberQueueWorkItemRepositoryItSpec extends PlaySpec
         savedWorkItem.item mustBe updatedWI.item
       }
 
+      // The database is storing these as ISODates and so truncated to milliseconds
```
Comment (mauro-fanton) [inline it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala:130]: just a thought, I wonder if we could use the timestampMillisecond every where here and so remove the other timestamp

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala (hunk 2 above)
Comment (ChrisCookOC) [reply it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala]: Done

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala (hunk 3 above)
Comment (ChrisCookOC) [reply it/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/ExciseNumberQueueWorkItemRepositoryItSpec.scala]: Done (if I've understood correctly.)

**Review by RobertLaverick — APPROVED** [review]

**Pull request merged**